### ISSUE TITLE
Pasting urls without selection still results in ()[url]

### ISSUE THREAD

**Issue by 06000208** [denolehov/obsidian-url-into-selection#15]
Me and phnx#9804 experienced this after updating the plugin. Happens whenever the plugin is enabled, stops happening when you disable & then restart obsidian.

Apologies if this is a duplicate issue or already known.

Mentioned in the obsidian discord [here](https://discord.com/channels/686053708261228577/694233507500916796/834331090097274890) and [here](https://discord.com/channels/686053708261228577/694233507500916796/834403624762736710)

![2021-04-21_08-04-39](https://user-images.githubusercontent.com/52764066/115558380-4c036c00-a278-11eb-9986-36c165b15f8c.gif)

**Comment by denolehov** [comment]
Hey, this will be fixed once #14 is merged :)

**Comment by AidenLx** [comment]
the fix is now a separate pr #17

**Comment by denolehov** [comment]
thank you @AidenLx for the fix! 

@06000208 it has been released, please update the plugin :)
